README.md

```diff
@@ -176,6 +176,8 @@ let turntable = Turntable(cassetteName: "dvr_single")
 
 That way you won't have to throw anything away.
 
+> Note: only use it for cassettes that you already have in the bundle, otherwise recording will crash trying to read missing file.
```

Comment:
Why is this as a quote? IMO it should be just regular text, although "Note:" could be bold.

Reply:
That's just what I always use for notes 🤷‍♂️ but I'm fine with bold too